draft-ietf-quic-transport.md

```diff
@@ -4578,7 +4578,8 @@ likely to compromise the packet protection for all 0-RTT packets because the
 same key and nonce could be used to protect different content.
 
 A client only receives acknowledgments for its 0-RTT packets once the handshake
-is complete.  Consequently, a server might expect 0-RTT packets to start with a
+is complete, as defined in the Handshake Complete section of {{QUIC-TLS}}.
```

Comment:
```suggestion
is complete, as defined Section 4.1.1 of {{QUIC-TLS}}.
```

Reply:
That's what I first wrote, but I was worried about section renumbering if TLS draft is edited. Fine with me.